.github/labeler.yml

```
@@ -40,16 +54,28 @@
 
 "Info / User Interface":
   - "**/ui/**/*"
+  - "**/ui**/*"
```

```suggestion
  - "src/**ui**"
```
Note that, this also thinks these files count as ui changes. I can't think of a better regex for this at the moment, file naming ain't helping it, this is probably fine for now
```
./src/butchery_requirements.cpp
./src/requirements.h
./src/handle_liquid.h
./src/butchery_requirements.h
./src/build_reqs.h
./src/requirements.cpp
./src/handle_liquid.cpp
```